### PR TITLE
add www for this

### PR DESCRIPTION
**Pull request by eliwilliamson**
i added a subdirectory for the website.

### CODE REVIEW

File: www/site/layouts/partials/header.html [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  <head>
+    <title>{{ .Site.Title }}</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="apple-touch-icon" sizes="180x180" href="/img/favicon/apple-touch-icon.png">
+    <link rel="icon" type="image/png" href="/img/favicon/favicon-32x32.png" sizes="32x32">
+    <link rel="icon" type="image/png" href="/img/favicon/favicon-16x16.png" sizes="16x16">
+    <link rel="manifest" href="/img/favicon/manifest.json">
+    <link rel="mask-icon" href="/img/favicon/safari-pinned-tab.svg" color="#2eb83c">
+    <meta name="apple-mobile-web-app-title" content="NetlifyCMS">
+    <meta name="application-name" content="NetlifyCMS">
```
Comment (calavera) [inline www/site/layouts/partials/header.html]: Wrong content

**Comment by calavera** [comment]
@eliwilliamson Please, add a section to the repository readme talking about the website and how to build it.

File: www/site/layouts/partials/header.html (hunk 1 above)
Comment (eliwilliamson) [inline www/site/layouts/partials/header.html]: same as other repo - on it 👍

**Comment by eliwilliamson** [comment]
@calavera the readme within www breaks it down - is this what you are looking for?
https://github.com/netlify/gotrue/blob/8c16a82f03cb0ddf4e2a44dc1de97ab969e2551e/www/README.md

File: www/netlify.toml [hunk 2]
```diff
@@ -0,0 +1,6 @@
+[build]
```
Comment (calavera) [inline www/netlify.toml]: @eliwilliamson this file needs to be in the root of the repository. It also has to include the next line under the `build` section to ensure we build from the www directory:

```
base = "www"
```

**Comment by calavera** [comment]
@eliwilliamson that readme works fine 👍 . There is still an issue remaining to be able to deploy the site from the subdirectory ☝️

**Comment by calavera** [comment]
https://www.gotrueapi.org/

**Pull request merged**